integreatly-docs/modules/ROOT/pages/index.adoc

```diff
@@ -63,14 +62,16 @@ include::_partials/ref_terminology.adoc[leveloffset=+1,tags={include-tags}]
 include::_partials/obtaining_firebase_cloud_messaging_credentials.adoc[leveloffset=+2,tags={include-tags}]
 include::_partials/enable_push_notifications_for_ios_app.adoc[leveloffset=+2,tags={include-tags}]
 [[setting-up-the-push-client-app]]
-== Configuring your client app
-//include::_partials/downloading-configuration-file.adoc[leveloffset=+2,tags={include-tags}]
-include::_partials/setting-up-the-push-sdk.adoc[leveloffset=+2,tags={include-tags}]
-include::_partials/registering_device.adoc[leveloffset=+1,tags={include-tags}]
-include::_partials/sending-a-push-notification.adoc[leveloffset=+1,tags={include-tags}]
-include::_partials/handling-push-notifications.adoc[leveloffset=+1,tags={include-tags}]
 
+== Configuring the Push Notification Service
+
+Access the Push Notification Service section by clicking *Push Notification Service* from the {webapp}. Log in if you are requested to do so.
 
-// Restore the context to what it was before this assembly.
-ifdef::parent-context[:context: {parent-context}]
-ifndef::parent-context[:!context:]
+* Click *Start Here*
+* Enter a name for your application and click *Create App*
+* Click *Add a Variant*
+* Enter a name for your variant and select the platform that intend to target (IOS/Android)
+* Enter the *Push Network* details necessary for your push network
+* Click *Create*
+* Code snippets for your applications are displayed.  Copy each snippet into the appropriate file in your application 
```

Comment:
the results of an action should be listed at the bottom of the task, then the next task starts with imperative